apps/cookbook/src/app/examples/checkbox-example/examples/reactive-forms.ts

```diff
@@ -0,0 +1,135 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+
+const config = {
+  selector: 'cookbook-checkbox-reactive-forms-example',
+  template: `<form [formGroup]="form">
+  <ng-container *ngFor="let checkbox of checkboxes">
+  <kirby-checkbox
+      formControlName="{{ checkbox.id }}"
+      [text]="checkbox.label"
+      [disabled]="!canSelectFavorite"
+    ></kirby-checkbox>
+  </ng-container>
+</form>
+<cookbook-example-configuration-wrapper>
+  <kirby-checkbox
+    [checked]="canSelectFavorite"
+    (click)="toggleCanSelectFavorite()"
+    text="Form field enabled"
+  ></kirby-checkbox>
+  
+  <section class="form-state">
+    <h4>Form state:</h4>
+    <p>
+      <strong>Value:</strong> {{ formValue| json }}
+      <br />
+      <strong>{{checkboxes[0].id}}:</strong>
+      <span [class.state-true]="form.get(checkboxes[0].id).valid">valid: {{ form.get(checkboxes[0].id).valid }}</span>
+      <span [class.state-true]="form.get(checkboxes[0].id).enabled">enabled: {{ form.get(checkboxes[0].id).enabled }}</span>
+      <span [class.state-true]="form.get(checkboxes[0].id).touched">touched: {{ form.get(checkboxes[0].id).touched }}</span>
+     <strong>{{checkboxes[1].id}}:</strong>
+     <span [class.state-true]="form.get(checkboxes[1].id).valid">valid: {{ form.get(checkboxes[1].id).valid }}</span>
+      <span [class.state-true]="form.get(checkboxes[1].id).enabled">enabled: {{ form.get(checkboxes[1].id).enabled }}</span>
+      <span [class.state-true]="form.get(checkboxes[1].id).touched">touched: {{ form.get(checkboxes[1].id).touched }}</span>
+      <strong>{{checkboxes[2].id}}:</strong>
+      <span [class.state-true]="form.get(checkboxes[2].id).valid">valid: {{ form.get(checkboxes[2].id).valid }}</span>
+      <span [class.state-true]="form.get(checkboxes[2].id).enabled">enabled: {{ form.get(checkboxes[2].id).enabled }}</span>
+      <span [class.state-true]="form.get(checkboxes[2].id).touched">touched: {{ form.get(checkboxes[2].id).touched }}</span>
+
+    </p>
+  </section>
+  </cookbook-example-configuration-wrapper>`,
+  codeSnippet: `checkboxes: { id: string; label: string }[] = [
+    { id: 'Bacon', label: 'Bacon' },
+    { id: 'Salami', label: 'Salami' },
+    { id: 'Tenderloin', label: 'Tenderloin' },
+  ];
+  
+  form: FormGroup;
+  canSelectFavorite = true;
+  formValue: any;
```

Comment:
I think we should clean this example up, so it is easier to understand.
Let us talk about this thursday!